internal/monitor/exporter.go

```diff
@@ -25,20 +24,72 @@ import (
 	"contrib.go.opencensus.io/exporter/ocagent"
 	"contrib.go.opencensus.io/exporter/prometheus"
 	"contrib.go.opencensus.io/exporter/stackdriver"
+	"github.com/googlecloudplatform/gcsfuse/v2/cfg"
+	"github.com/googlecloudplatform/gcsfuse/v2/common"
 	"github.com/googlecloudplatform/gcsfuse/v2/internal/logger"
 	"go.opencensus.io/stats/view"
 )
 
-var stackdriverExporter *stackdriver.Exporter
+func maybeStartStackdriverExporter(exporterIntervalSecs int64) common.ShutdownFn {
+	if exporterIntervalSecs <= 0 {
```

Comment:
do you want to add some info logs here as well to know why the exporter hasn't started?